src/shared/tpm2-util.c

```diff
@@ -1026,11 +1026,11 @@ static int tpm2_pcr_read(
                 const TPML_PCR_SELECTION *pcr_selection,
                 TPML_PCR_SELECTION *ret_pcr_selection,
                 TPM2B_DIGEST **ret_pcr_values,
-                size_t *ret_pcr_values_size) {
+                size_t *ret_pcr_values_count) {
```

Comment:
btw, usually we prefix an "n_" to variables that contain counts of things, rather that suffix with "_count".

i.e.:

```c
struct my_object *objects;
size_t n_objects;
```

hence, maybe name this field `ret_n_pcr_values`?

doesn#t matter though